src/main/java/com/kodilla/ecommercee/domain/OrderDto.java

```diff
@@ -1,14 +1,22 @@
 package com.kodilla.ecommercee.domain;
 
+import com.kodilla.ecommercee.User;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 
+import java.util.ArrayList;
+import java.util.List;
+
 
 @Getter
 @NoArgsConstructor
 @AllArgsConstructor
 public class OrderDto {
     private Long id;
+    private int orderNumber;
     private String productGroup;
+    private int price;
```

Comment:
Wydaje mi się, że cena powinna być typu double, chyba, że nie rozmieniamy się na drobne :)

Reply:
jeśli byłby to bank to rekomendowałbym BigDecimal. tutaj może jednak precyzja double wystarczy

Reply:
Ustawiłem BigDecimala